text_generation/causal_lm/cpp/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2023-2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
do we need it?

Reply:
Same as above, investigated how to link TBB. Removed.